projects/1.18/assets/little-logistics/zh_cn.json

```
@@ -0,0 +1,104 @@
+{
+  "item.littlelogistics.tug": "蒸汽拖船",
+  "item.littlelogistics.barge": "货柜驳船",
+  "item.littlelogistics.chunk_loader_barge": "区块加载驳船",
+  "item.littlelogistics.seater_barge": "坐式驳船",
+  "item.littlelogistics.fishing_barge": "自动捕鱼驳船",
+  "item.littlelogistics.spring": "车链",
+  "item.littlelogistics.tug_route": "拖船航线",
+  "item.littlelogistics.locomotive_route": "机车路线",
+  "item.littlelogistics.energy_tug": "能源拖船",
+  "item.littlelogistics.fluid_barge": "液舱驳船",
+  "item.littlelogistics.fluid_car": "液体罐车",
+  "item.littlelogistics.chest_car": "货柜车厢",
+  "item.littlelogistics.chunk_loader_car": "区块加载车厢",
+  "item.littlelogistics.seater_car": "车厢",
+  "item.littlelogistics.steam_locomotive": "蒸气火车头",
+  "item.littlelogistics.energy_locomotive": "能源火车头",
```

```suggestion
  "item.littlelogistics.steam_locomotive": "蒸汽机车",
  "item.littlelogistics.energy_locomotive": "电力机车",
```
```suggestion
  "item.littlelogistics.steam_locomotive": "蒸汽火车头",
  "item.littlelogistics.energy_locomotive": "电力火车头",
```
`蒸汽`还是`蒸气`？
`电力`可能会好一点，`火车头`还需再讨论

---

entity部分也应相应做出修改

---

> “蒸汽机”是利用蒸汽在汽缸内膨胀，推动活塞运动而产生动力的一种发动机，使用的是水蒸气，“蒸汽机”中的“汽”不宜写成“气”。

> “蒸汽锤”指靠水蒸气产生动力的机器锤。“蒸汽锤”也简称为“汽锤”。“汽锤”跟“气锤”不同，“气锤”也叫“空气锤”，指利用压缩空气产生动力的机器锤。(《语言文字报》原主编 杜永道)

http://www.hwjyw.com/resource/content/2011/12/27/22947.shtml

---

火车头建议改成机车